script.js

```diff
@@ -75,12 +109,32 @@ function setup() {
   episodesArticle.className = "episodes";
 
   //------Showing all the episodes
-  episodes.forEach(obj => {
-    let newDiv = document.createElement("div");
-    newDiv.innerHTML = `<h1>S${String(obj.season).padStart(2,0)}E${String(obj.number).padStart(2,0)}</h1><div><img src=${obj.image.medium} /></div><span>Hover for a summery</span><div class='overlay'>${obj.summary}</div>`;
-    newDiv.className = "episode";
-    episodesArticle.appendChild(newDiv);
-  });
+  const episodesHandler = (arr) => {
+
+    while (episodesArticle.hasChildNodes()) {
+      episodesArticle.removeChild(episodesArticle.lastChild);
+
+    }
+    if (arr.length === 0) {
+
+      return episodesArticle.innerHTML = "<h1 style='text-align:center ; width:100%'>Ooops, Sorry nothing found!</h1>"
+    }
+    arr.forEach(obj => {
+
+      let newDiv = document.createElement("div");
+      if (arr.length === 0) {
+        document.querySelector(".episodes")
+      } else {
+        newDiv.innerHTML = `<h1>S${String(obj.season).padStart(2,0)}E${String(obj.number).padStart(2,0)}</h1><div><img src=${obj.image.medium} /></div><span>${obj.name}</span><div class='overlay'>${obj.summary}</div>`;
+        newDiv.className = "episode";
```

Comment:
Very good use of padStart! However, the code is a little hard to read. I would recommend installing Prettier if you don't have it.